tests/performance/common/common.go

```diff
@@ -63,6 +69,8 @@ func HandleCloudDeployment(cloudConfigMap, cloudCoreDeployment, apiserver2, conf
 	//Handle cloudCore deployment
 	go utils.HandleDeployment(true, false, http.MethodPost, deploymenthdl, cloudCoreDeployment, imgURL, "", cloudConfigMap, 1)
 
+	CloudCoreDeployment = cloudCoreDeployment
```

Comment:
i feel you are assigning double time to CloudCoreDeployment var,  CloudCoreDeployment been assingned the deployment name while generating random name for cloudcore deployment. CloudCoreDeployment has global scope.

you can remove this stpe.

Reply:
Oh... I see. will add this part in the hubtest_suite

Reply:
Done:)